packages/server/lib/controllers/v1/logs/searchLogs.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from 'zod';
+import { asyncWrapper } from '../../../utils/asyncWrapper.js';
+import { requireEmptyQuery, zodErrorToHTTP } from '../../../utils/validation.js';
+import type { SearchLogs } from '@nangohq/types';
+import { model, envs } from '@nangohq/logs';
+
+const validation = z
+    .object({
+        limit: z.number().optional().default(100)
+    })
+    .strict();
+
+export const searchLogs = asyncWrapper<SearchLogs>(async (req, res) => {
+    if (!envs.NANGO_LOGS_ENABLED) {
+        res.status(422).send({ error: { code: 'feature_disabled' } });
```

Comment:
404. no?

Reply:
yeah I wasn't sure because 404 means it the URL is invalid (basically), it's kind of invalid but also not and doesn't convey the same initial message. Happy to put 404

Reply:
404 means resource doesn't exist so it fits quite well I think. The error message conveys that is a feature that can be enabled

Reply:
changed 👌🏻